docs/guide/front-end-helpers.md

```diff
@@ -182,7 +182,7 @@ There is also a `runPlayers` method on `apos.util`. That is run for us using `ap
 
 ## HTTP request methods
 
-These are all available in the browser on `apos.http`, e.g., `apos.http.get('/api/v1/article')`. They include for specific HTTP request method utilities as well as other helpers for making HTTP requests.
+These are all available in the browser on `apos.http`, e.g., `apos.http.get('/api/v1/article')`. TThey include utility methods to make requests with each HTTP method (GET, POST,PATCH, PUT and DELETE) as well as other helpers for making HTTP requests.
```

Comment:
[SORRY]Missing space between POST, and PATCH[/SORRY]

Reply:
that was a test. you passed.